.circleci/config.yml

```diff
@@ -202,7 +202,8 @@ jobs:
     parallelism: 1
     steps:
       - prepare
-      - run: npx lerna run prepublishOnly --scope '@automattic/jetpack-blocks'
+      - run: npx lerna bootstrap --concurrency=2 --scope '@automattic/jetpack-blocks'
```

Comment:
Equivalent of `npm install` for a monorepo package. Since the package is not required from the root, it requires an independent install.

Reply:
I'm giving this the side-eye. What npm commands does it end up running under the hood? What locks does it rely on or modify?

Reply:
Good question and I don't know to be honest. The issue I was adressing here is that with this package totally detached from the root (root doesn't depend on it) `npm install` no longer installs its dependencies.

I'd love to hear more informed alternatives 🙂

Reply:
Naive question: Can we `cd packages/jetpack-blocks && npm run prepublishOnly`?

Reply:
Uh nvm that's what you're doing on the line below :man_facepalming: